src/Analyzers/CSharp/Tests/MakeStatementAsynchronous/CSharpMakeStatementAsynchronousCodeFixTests.cs

```diff
@@ -2,226 +2,240 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 using System.Threading.Tasks;
-using Microsoft.CodeAnalysis.CodeFixes;
-using Microsoft.CodeAnalysis.CSharp;
 using Microsoft.CodeAnalysis.CSharp.CodeFixes.MakeStatementAsynchronous;
-using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Editor.UnitTests.CodeActions;
 using Microsoft.CodeAnalysis.Test.Utilities;
+using Microsoft.CodeAnalysis.Testing;
 using Xunit;
-using Xunit.Abstractions;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Diagnostics.MakeStatementAsynchronous
 {
+    using VerifyCS = CSharpCodeFixVerifier<
+        EmptyDiagnosticAnalyzer,
+        CSharpMakeStatementAsynchronousCodeFixProvider>;
+
     [Trait(Traits.Feature, Traits.Features.CodeActionsMakeStatementAsynchronous)]
-    public class CSharpMakeStatementAsynchronousCodeFixTests : AbstractCSharpDiagnosticProviderBasedUserDiagnosticTest
+    public class CSharpMakeStatementAsynchronousCodeFixTests
     {
-        public CSharpMakeStatementAsynchronousCodeFixTests(ITestOutputHelper logger)
-           : base(logger)
-        {
-        }
-
-        internal override (DiagnosticAnalyzer, CodeFixProvider) CreateDiagnosticProviderAndFixer(Workspace workspace)
-            => (null, new CSharpMakeStatementAsynchronousCodeFixProvider());
-
-        private static readonly TestParameters s_asyncStreamsFeature = new TestParameters(parseOptions: new CSharpParseOptions(LanguageVersion.CSharp8));
-
         [Fact]
         public async Task FixAllForeach()
         {
-            await TestInRegularAndScript1Async(
-IAsyncEnumerable + @"
+            await new VerifyCS.Test()
+            {
+                TestCode = @"
 class Program
 {
-    void M(System.Collections.Generic.IAsyncEnumerable<int> collection)
+    async void M(System.Collections.Generic.IAsyncEnumerable<int> collection)
```

Comment:
I'd honestly prefer you not change test inputs. It's really not necessary, and it does demonstrate the behavior of these features in these scenarios.

Reply:
So you would actually preffere to delete all `async`s and have 2 compilation errors per test output?